astropy/table/table.py

```diff
@@ -2556,10 +2555,13 @@ def write(self, *args, **kwargs):
           >>> dat = Table([[1, 2], [3, 4]], names=('a', 'b'))
           >>> dat.write('table.dat', format='ascii')
 
-        The arguments and keywords (other than ``format``) provided to this function are
-        passed through to the underlying data reader (e.g. `~astropy.io.ascii.write`).
+        The arguments and keywords (other than ``format`` and
+        ``serialize_methods``) provided to this function are passed through to
+        the underlying data reader (e.g. `~astropy.io.ascii.write`).
```

Comment:
Could we add a `Parameters` section here that describes `format` and `serialize_method`?

Reply:
Excellent idea, and one that I applied to `Table.read` as well.